### PR TITLE
radiusArc: due to float looseness the length computation can be sligh…

### PR DESCRIPTION
**Pull request by diorcety**
…tly over radius length, leading to a ValueError exception. Using TOL value in order to remove this issue

### CODE REVIEW

**Comment by jmwright** [comment]
@adam-urbanczyk and @lorenzncode Any thoughts on this PR?

**Comment by lorenzncode** [comment]
It's probably reasonable.  

```py
res = cq.Workplane().radiusArc((0, 2 + 0.999999e-6), 1)
```

This fails in master, allowed with PR by automatically increasing the radius from 1 to ~1+TOL/2 where TOL is 1e-6 in cq.py.

File: cadquery/cq.py [hunk 1]
```diff
@@ -2198,7 +2198,10 @@ def radiusArc(
         # Calculate the sagitta from the radius
         length = endPoint.sub(startPoint).Length / 2.0
         try:
-            sag = abs(radius) - math.sqrt(radius ** 2 - length ** 2)
+            sag = abs(radius)
+            r_2_l_2 = radius ** 2 - length ** 2
```
Comment (adam-urbanczyk) [inline cadquery/cq.py:2202]: Maybe add a comment explaining why this is needed

Comment (diorcety) [reply cadquery/cq.py:2202]: Comment added

**Comment by adam-urbanczyk** [comment]
LGTM, shall I merge?

**Comment by jmwright** [comment]
+1 to merge

**Comment by adam-urbanczyk** [comment]
Thanks @diorcety

**Pull request merged**